cmd/snap/cmd_moo.go

```diff
@@ -0,0 +1,37 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type cmdMoo struct{}
+
+func init() {
+	cmd := addCommand("moo", "", "", func() flags.Commander { return &cmdMoo{} }, nil, nil)
+	cmd.hidden = true
+}
+
+func (cmd cmdMoo) Execute(args []string) error {
+	return fmt.Errorf(`unknown command "moo", try "apt moo"`)
```

Comment:
```console
$ apt moo
Ubuntu Core does not use apt-get, see 'snap --help'!
```

Reply:
how could that have been missed for so long !!!

Reply:
It could fail with `we require more apt powers`